packages/generator/src/v1.1/generateMetadataFile.ts

```diff
@@ -39,10 +40,7 @@ export async function generateMetadata(
         userAgent: "foundry-typescript-osdk/0.0.1",
     },
     objects: {
-        ${
-      objectNames.map((name) => `${name}: ${name},`)
-        .join("\n")
-    }
+        ${commaSeparatedIdentifiers(objectNames)}
```

Comment:
This is a behavior change from the fully spelled out keys to the shorthand. I think that's fine and what you intended.